cmd/multiclusterapp.go

```diff
@@ -138,17 +162,69 @@ func MultiClusterAppCommand() cli.Command {
 						Name:  "set",
 						Usage: "Set answers for the template, can be used multiple times. Example: --set foo=bar",
 					},
+					cli.StringSliceFlag{
+						Name: "role,r",
+						Usage: "Set roles required to launch/manage the apps in target projects. Specified roles on " +
+							"upgrade will override all the original roles. Leave it empty to keep current roles",
```

Comment:
Why are roles on upgrade overriding all existing roles? If user has to add more roles will they be expected to provide all existing roles as input again? If so then can we add that as well to the usage?

Reply:
It's a simpler way to update roles. Or we will need to provide different flags/commands to add or delete roles of a mcapp. Updated the usage.